apim.tf

```
@@ -1,32 +1,23 @@
-locals {
-  app_name      = "pre-ams-integration"
-  function_name = "CheckBlobExists"
-  env_to_deploy = var.env == "sbox" ? 1 : 0
-}
-
 module "ams_product" {
-  count                 = local.env_to_deploy
   source                = "git@github.com:hmcts/cnp-module-api-mgmt-product?ref=master"
   api_mgmt_name         = "sds-api-mgmt-${var.env}"
   api_mgmt_rg           = "ss-${var.env}-network-rg"
   approval_required     = false
-  name                  = local.app_name
+  name                  = "pre"
   published             = true
   subscription_required = false
 }
 
-#WIP used toffee for test purposes
 module "ams_api" {
```

wouldn't be ams api anymore so just api maybe?
```suggestion
module "api" {
```